en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+    
+- **Error request count**
+
+    ![Error request count](../assets/img/insights/overview-page-error-request-count.png){.cInlineImage-half}
+
+    This widget displays the total number of requests that have resulted in errors in your selected environment during the selected time range.
+    
+- **Average Error Rate**
+
+    ![Average error rate](../assets/img/insights/overview-page-error-rate.png){.cInlineImage-half}
+
+    This widget displays the average error rate (error count/total request count) in your selected environment for the selected time interval. You can use this widget as an indicator to know the health of the system. If the error rate is high, investigate further using the Viewing API Analytics on Errors. 
+
+- **95th Percentile Latency**
+
+    ![95th percentile latency](../assets/img/insights/overview-page-error-rate.png){.cInlineImage-half}
+
+    This widget displays the 95th percentile of all API latencies in your selected environment for the selected time interval. You can use this widget to know whether the whole system operates under given SLAs. This metric gives the first indication of slow APIs. To investigate further, see Viewing API Analytics on Latency. 
+
+- **API Request Summary**
+
+    ![API request summary](../assets/img/insights/overview-page-timeline.png){.cInlineImage-full}
+
+    This chart combines the above three widgets and plots them in a timeline. The y-axis on the left displays the request count and the error count. The x-axis shows time, and the y-axis on the right shows the latency in milliseconds. The granularity of the data points is decided based on the time range you have selected. The tooltip provides the exact value of all three metrics accurately.
+
+### Traffic
+
+This page shows information related to the traffic that goes through your API management system. This includes API usage, application usage, resource usage, etc. You can use this page to investigate the usage of APIs and applications, traffic patterns, etc.
+
+![Traffic](../assets/img/insights/traffic-page-full.png){.cInlineImage-full}
+
+You can filter the information displayed in the widgets as follows:
+
+| **Filtering Option** | **Description** |
+|--------------------|-------------------------------------------------------------------------------------------------|
+| **By API**         | In the **API** field, you can select the APIs for which you want to view analytics. **All** is selected by default. You can select one or multiple APIs from this selector and view the aggregated result. Once you select an API, you can further filter by a specific application that uses the selected API via the **Application** field described below. |
```

Comment:
Shall we combine the two sentences as follows: "In the **API** field, you can select one or more APIs for which you want to view analytics."
Is application filtering a second level filtering?

Reply:
Done